src/foraging_gui/Foraging.py

```diff
@@ -147,6 +153,31 @@ def __init__(self, parent=None,box_number=1,start_bonsai_ide=True):
             self._ReconnectBonsai()   
         logging.info('Start up complete')
     
+    def _load_rig_metadata(self):
+        '''Load the latest rig metadata'''
+        if not os.path.exists(self.rig_metadata_folder):
+            self._manage_warning_labels(self.MetadataWarning,warning_text='No rig metadata found!')
```

Comment:
We should log an error if the rig metadata folder is not found, and this warning should specify that the folder is not found

Reply:
Updated